[DTM] Add application requirements (#973)
SoftwareAG/c8y-docs: content/dtm/dtm-introduction.md

The home screen of the DTM application gives an overview of the application. You can see the number of asset models, asset properties and assets available at a glance. You can add asset properties, asset models and assets using the quick links.

{{< c8y-admon-req >}}

> * Your tenant must be subscribed to both Digital twin manager web application and Dtm-ms microservice.

Reviewer: ```suggestion
* Your tenant must be subscribed to the Digital Twin Manager application and the Dtm-ms microservice.
```

> * Your tenant must be subscribed to both Digital twin manager web application and Dtm-ms microservice.

Reviewer: Dtm-ms > is this the exact spelling?
